app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -137,7 +137,7 @@ data class Status(
         )
     }
 
-    private fun getEditableText(): String {
+    fun getEditableText(): String {
```

Comment:
No, don't use this. This is only here to have a fallback for ancient Mastodon versions before deleting returned the status source. To get the status source without deleting a status, call `GET /api/v1/statuses/:id/source`.